tests/data/errorsheet.csv

```diff
@@ -0,0 +1,5 @@
+sample,insdc_accession
```

Comment:
This file seems to have the wrong heading (should be `error_accession`).

Reply:
Actually, wait. This is an example input samplesheet right, and not the output errors CSV file? In that case this is actually the correct heading/column name (`insdc_accession`). Sorry.